apps/docs-snippets2/src/provider/functionality-get-coins.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { Provider, Wallet } from 'fuels';
+
+import { LOCAL_NETWORK_URL, WALLET_PVT_KEY } from '../env';
+
+const provider = await Provider.create(LOCAL_NETWORK_URL);
+const wallet = Wallet.fromPrivateKey(WALLET_PVT_KEY, provider);
```

Comment:
If we want the snippet to be copy/pasteable shouldn't these imports be included in the `getCoins-1` region?

Reply:
[16b1343](https://github.com/FuelLabs/fuels-ts/pull/3357/commits/16b1343715ac3718e060a748b98ea7030c0550ca)